ext/ffi/callback.rs

```diff
@@ -174,7 +160,10 @@ unsafe fn do_ffi_callback(
   // refer the same `let bool_value`.
   let mut cb_scope = v8::CallbackScope::new(context);
   let scope = &mut v8::HandleScope::new(&mut cb_scope);
-  let func = callback.open(scope);
+  let func = std::mem::transmute::<
```

Comment:
The magic above always bugged me a bit, so this is good to see. 👍

Reply:
Looking back on it, I'm quite happy to get rid of isolate pointers as well :)